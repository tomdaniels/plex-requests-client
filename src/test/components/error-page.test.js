import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/error-page';

test('should render the 404 page, link back to dash', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});