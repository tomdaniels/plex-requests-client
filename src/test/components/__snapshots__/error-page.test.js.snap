// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the 404 page, link back to dash 1`] = `
<div>
  404! 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
